test/e2e/bots_test.go

```diff
@@ -48,18 +48,22 @@ type Config struct {
 	Discord     DiscordConfig
 }
 
+type AttachmentStatus = map[config.Level]string
+
 type SlackConfig struct {
-	BotName                  string `envconfig:"default=botkube"`
-	TesterName               string `envconfig:"default=tester"`
-	AdditionalContextMessage string `envconfig:"optional"`
+	BotName                  string           `envconfig:"default=botkube"`
+	TesterName               string           `envconfig:"default=tester"`
+	AdditionalContextMessage string           `envconfig:"optional"`
+	AttachmentStatus         AttachmentStatus `envconfig:"optional"`
 	TesterAppToken           string
 	MessageWaitTimeout       time.Duration `envconfig:"default=30s"`
 }
 
 type DiscordConfig struct {
-	BotName                  string `envconfig:"default=botkube"`
-	TesterName               string `envconfig:"default=botkube_tester"`
-	AdditionalContextMessage string `envconfig:"optional"`
+	BotName                  string           `envconfig:"default=botkube"`
+	TesterName               string           `envconfig:"default=botkube_tester"`
+	AdditionalContextMessage string           `envconfig:"optional"`
+	AttachmentStatus         AttachmentStatus `envconfig:"optional"`
```

Comment:
we already have dedicated methods on each driver, there is no need to pass it via config :)
```suggestion
```